content/source/docs/enterprise/install/installer.html.md

```diff
@@ -52,6 +52,18 @@ This change updates the proxy settings for the Terraform Enterprise application
 `NOTE: Please take precautions on application outage when applying configuration change, i.e. wait for all runs to finish, prevent new runs to trigger`
 4. Restart the Replicated services following [the instructions for your distribution](https://help.replicated.com/docs/native/customer-installations/installing-via-script/#restarting-replicated).
 
+### Proxy and Service Discovery Process
+
+Workspaces in Terraform Enterprise can be executed in various methods according to selected workflows and for each selected workflow, it uses **remote backend** to perform [remote operation](https://www.terraform.io/docs/cloud/run/#remote-operations) unless **Execution Mode** of the workspace is set **Local**. Terraform CLI that is executed as part of the **remote operation** inside Terraform Enterprise follows the same standard process to perform [service discovery](https://www.terraform.io/docs/internals/remote-service-discovery.html) where Terraform Enterprise itself is the remote service for Terraform CLI, depends on your infrastructure setup, the fully qualified name of Terraform Enterprise may be required to include into **Proxy ByPass** configuration of the Terraform Enterprise application layer to allow Terraform CLI to communicate to Terraform Enterprise itself, to configure the proxy bypass address of Terraform Enterprise application, navigate to port **8800** under `/console/settings`:
```

Comment:
```suggestion
Unless the **Execution Mode** of a workspace is set **Local**, Terraform Enterprise executes [remote operations](https://www.terraform.io/docs/cloud/run/#remote-operations) and uses a **remote backend**. 

The Terraform CLI performs **remote operations** for Terraform Enterprise and uses [service discovery](https://www.terraform.io/docs/internals/remote-service-discovery.html) to find the Terraform Enterprise service itself. Depending on your infrastructure setup, you may need to include the fully qualified name of Terraform Enterprise in the **Proxy ByPass** configuration of Terraform Enterprise's application layer. 

Configure the proxy bypass address of Terraform Enterprise application on port **8800** on the path `/console/settings`:
```

Reply:
Also please check this suggestion carefully. I'm not sure I 100% understood the text so I took my best guess.

Reply:
I'm going to synthesize Judith's edits with what I think I'm reading in the original:

```suggestion
Unless the execution mode of a workspace is set to "local", Terraform Enterprise performs [remote operations](/docs/cloud/run/index.html#remote-operations), running Terraform in its own worker VMs. 

When running within Terraform Enterprise's worker VMs, Terraform uses [service discovery](/docs/internals/remote-service-discovery.html) to find the Terraform Enterprise service itself. Depending on your infrastructure setup, you may need to tell Terraform Enterprise not to access its own hostname via the proxy, so that Terraform can communicate with the Terraform Enterprise services.

To do this, add Terraform Enterprise's fully qualified hostname to the **Proxy Bypass** setting in Terraform Enterprise's dashboard. The proxy bypass setting can be found on port **8800** on the path `/console/settings`.
```